### PR TITLE
Tlf 188

### PR DESCRIPTION
**Pull request by hellomatiHO**
What:
Following a review of LCS, added the initial folders bin and kube
Created the drone.yml file

Why:
The files are needed for the app to be able to build the artifact
The pipeline file declares the steps for the CI/CD

### CODE REVIEW

File: kube/configmaps/configmaps.yml
Comment (gadorcruz) [inline kube/configmaps/configmaps.yml:1]: The right file is configmap.yml so the configmaps.yml is not needed.

**Review by gadorcruz — APPROVED** [review]

**Comment by adityababumallisettiHO** [comment]
Can we also add those unfixable which are part of hof packages and without latest direct updates to packages whitelisted in .snyk file. so that it passes all the three checks please

**Comment by gadorcruz** [comment]
I would not like to whitelist the vulnerabilities as the point is to review them first:
https://collaboration.homeoffice.gov.uk/display/DSASS/Handling+Failed+Snyk+Scans+in+Drone+Pipelines

**Comment by dk4g** [comment]
> Can we also add those unfixable which are part of hof packages and without latest direct updates to packages whitelisted in .snyk file. so that it passes all the three checks please

I agree with @gadorcruz that the SNYK report needs to be reviewed and the corresponding JIRA ticket will be logged for each ignore item if it has already been logged.

This will be done as part of the development process, separate ticket will be raised to cover the ignore list.

File: kube/app/deployment.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  {{ if eq .KUBE_NAMESPACE .PROD_ENV }}
+  name: {{ .APP_NAME }}
+  {{ else if eq .KUBE_NAMESPACE .BRANCH_ENV }}
+  annotations:
+    downscaler/uptime: {{ .NON_PROD_AVAILABILITY }}
```
Comment (dk4g) [inline kube/app/deployment.yml:9]: `NON_PROD_AVAILABILITY` is missing in drone.yml

**Review by dk4g — APPROVED** [review]

**Pull request merged**